specifyweb/frontend/js_src/lib/components/FormCells/index.tsx

```diff
@@ -189,33 +187,22 @@ const cellRenderers: {
       return null;
     const pickList = toTable(currentResource, 'PickList');
 
-    if (typeof pickList === 'object' && showPickListForm)
-      return <PickListEditor relationship={relationship} resource={pickList} />;
-    else if (interactionCollection === false || actualFormType === 'form')
-      return (
-        <SubView
-          formType={actualFormType}
-          icon={icon}
-          isButton={isButton}
-          mode={mode}
-          parentFormType={parentFormType}
-          parentResource={currentResource}
-          relationship={relationship}
-          sortField={sortField}
-          viewName={viewName}
-        />
-      );
-    else
-      return (
-        <FormTableInteraction
-          collection={interactionCollection}
-          dialog={false}
-          mode={mode}
-          sortField={sortField}
-          onClose={f.never}
-          onDelete={undefined}
-        />
-      );
+    return typeof pickList === 'object' && showPickListForm ? (
+      <PickListEditor relationship={relationship} resource={pickList} />
+    ) : (
+      <SubView
+        formType={actualFormType}
+        icon={icon}
+        isButton={isButton}
+        isInteraction={interactionCollection !== false}
```

Comment:
Good catch with the fact that fetching interactionCollection again was redundant.

Since you are no longer using the actual value of the interactionCollection, you don't have to fetch it anymore.
Go ahead and replace the useAsyncState with something much simpler.

<details><summary>Solution</summary>

```tsx
    const isInteraction = typeof relationship === 'object' &&
          relationshipIsToMany(relationship) &&
          typeof data?.resource === 'object' &&
          [
            'LoanPreparation',
            'GiftPreparation',
            'DisposalPreparation',
          ].includes(relationship.relatedModel.name);
```
<p>



</p>
</details> 

And now that it's no longer async, you don't have to do place it in here and can instead place it much closer to where it's used (probably in `IntegratedRecordSelector`)